src/honey_badger/honey_badger.rs

```diff
@@ -34,6 +35,8 @@ pub struct HoneyBadger<C, N: Rand> {
     pub(super) rng: Box<dyn Rng + Send + Sync>,
     /// Represents the optimization strategy to use for output of the `Subset` algorithm.
     pub(super) subset_handling_strategy: SubsetHandlingStrategy,
+    /// The schedule for which rounds we should use threshold encryption.
+    pub(crate) encryption_schedule: EncryptionSchedule,
```

Comment:
I'd prefer making this `pub(super)`, like the other fields, and adding a getter for it:
The idea is that the algorithm modules can be used independently, and the user should be able to implement all of the `DynamicHoneyBadger` functionality by themselves, if needed, outside of this crate.